dace/sdfg/graph.py

```diff
@@ -207,15 +207,15 @@ def add_nodes_from(self, node_list):
         for node in node_list:
             self.add_node(node)
 
-    def node_id(self, node):
+    def node_id(self, node: Any) -> int:
         """Returns a numeric ID that corresponds to the node index in the
            internal graph representation (unique)."""
         for i, n in enumerate(self.nodes()):
             if node == n:
                 return i
         raise NodeNotFoundError(node)
 
-    def edge_id(self, edge):
+    def edge_id(self, edge: Edge) -> int:
```

Comment:
why Edge is not inside the module (e.g. dace.sdfg,Edge)?

Reply:
it exists in the same file (`dace.sdfg.graph.Edge`).